.github/workflows/neuron-ci.yml

```diff
@@ -0,0 +1,243 @@
+name: NEURON CI
+
+on:
+  push:
+    branches: 
+      - master
+      - live-debug*
+  pull_request:
+    branches:
+      - master
+
+env:
+  BUILD_TYPE: Release
+  DEVELOPER_DIR: /Applications/Xcode_11.3.1.app/Contents/Developer
+  DESIRED_CMAKE_VERSION: 3.8.2
```

Comment:
global value for minimal cmake version

Reply:
yes, I had a bit of hard time to set it up and remembered you also struggled a bit. Action `name: Setup cmake` will install cmake with this version. I've kept the check of DESIRED vs ACTUAL in the main job however.